tests/warmup-manifest-build.yml

```diff
@@ -1,21 +1,2 @@
-version: 1.0
-jobs:
-  - type: asset
-    fetch: http
-    username: NEXUS_USERNAME
-    password: NEXUS_PASSWORD
-    source: https://devtools.jahia.com/nexus/service/local/artifact/maven/redirect?r=jahia-snapshots&g=org.jahia.modules&a=graphql-dxm-provider&v=LATEST
-    filepath: /tmp/graphql-dxm-provider-LATEST.jar
-  - type: module
-    id: graphql-dxm-provider
-    filepath: /tmp/graphql-dxm-provider-LATEST.jar
-  - type: module
-    id: sitemap
-    filepath: /tmp/artifacts/sitemap-SNAPSHOT.jar
-  - type: webproject
-    source: prepackaged
-    sitekey: digitall    
-  - type: modulesite
-    action: enable
-    moduleId: sitemap
-    siteId: digitall
+- include: mvn:org.jahia.packages/jahia-additional-modules/JAHIA_VERSION/yaml/provisioning
```

Comment:
Shouldn't we rename the file to "provisioning-..." ?

Reply:
Sure, why not.